todo-app/src/ToDoStatus.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import './todo.css'
+import updateStatus from './toDos-DB.js'
+
+export default class ToDoStatus extends React.Component {
+    constructor(props) {
+        super(props)
+        this.status = {
+            index: this.props.index,
+            toDoStatus: this.props.toDoStatus
+        }
+    }
+    toggleCheckbox = () => {
+        if (this.refs.chxToDoStatus.checked !== true) {
+            updateStatus(this.status.index, "Not Done")
+            this.setState({
+                key: this.props.key,
+                toDoStatus: "Not Done"
+            }, () => { this.refs.valueToDoStatus.innerHTML = "( Not Done )" })
+        } else {
+            updateStatus(this.status.index, "Done")
+            this.setState({
+                index: this.props.key,
+                toDoStatus: "Done"
+            }, () => { this.refs.valueToDoStatus.innerHTML = "( Done )" })
+        }
+    }
+    componentDidMount() {
+        this.refs.chxToDoStatus.checked = (this.status.toDoStatus === "Done") ? true : false;
+    }
+    render() {
+        return (
+            <div className="toDo-Status-chx">
+                <div>
+                    <input
+                        type="checkbox"
+                        ref="chxToDoStatus"
```

Comment:
We should avoid using `ref`s in React. The way to accomplish this is by using a combination of `onChange` and `value` on the input, assuming that `status` is a boolean value. 

```js
export let ToDos = [
    {
        todoDesc: "Get Out of Bed",
        toDoStatus: true
    },
    {
        todoDesc: "Brush Teeth",
        toDoStatus: false
    },
    {
        todoDesc: "Eat Breakfast",
        toDoStatus: false
    }
];
```

For example:

```jsx
// Somewhere in the constructor 
this.state = {
  toDoStatus: this.props.toDoStatus
};

// Somewhere in render
<input
  type="checkbox"
  checked={this.state.toDoStatus}
  onChange={this.toggleCheckbox}
/>
```

In this case `toggleCheckbox` will be called with an `event` that contains the target with the new `checked` value, `event.target.checked`. Now that we have this value in the `toggleCheckbox` function, how can we modify the state without using `updateStatus` (basically we should remove this function 😄)??